.pipelines/pipeline.yaml

```diff
@@ -367,6 +367,14 @@ stages:
       os_version: 'ltsc2022'
       scaleup: 100
 
+  - template: singletenancy/dualstack-overlay/dualstackoverlay-e2e-job-template.yaml
+    parameters:
+      name: "dualstackoverlay_e2e"
+      displayName: AKS DualStack Overlay
+      pipelineBuildImage: "$(BUILD_IMAGE)"
+      testDropgz: ""
+      clusterName: "dualstackoverlaye2e"
```

Comment:
I think you will need to pull in the changes @jpayne3506   added to his PR ?

Reply:
We depend on if 2077 gets merged before this.